app/models/organization.rb

```diff
@@ -14,11 +14,19 @@ class Organization < ApplicationRecord
 
   validates :name, presence: true, uniqueness: true
 
+  def agents
```

Comment:
I believe all of these methods can be turned into scopes for a cleaner look and easier usage. E.g.

```ruby
has_many :agents, ->(o) { where.not(id: o.administrator_id) }, class_name: "User", dependent: :destroy
```